docs/source/index.rst

```diff
@@ -661,6 +661,7 @@ Flax), PyTorch, and/or TensorFlow.
     model_doc/mbart
     model_doc/megatron_bert
     model_doc/megatron_gpt2
+    model_doc/mluke
```

Comment:
This file is going to be removed and is not used anymore. You should add your model [here](https://github.com/huggingface/transformers/blob/master/docs/source/_toctree.yml)

Reply:
Oh, I see. Thank you for the pointer!